src/manifest.json

```diff
@@ -5,12 +5,11 @@
   "incognito": "not_allowed",
   "description": "Multi-Account Containers helps you keep all the parts of your online life contained in different tabs. Custom labels and color-coded tabs help keep different activities — like online shopping, travel planning, or checking work email — separate.",
   "icons": {
-    "48": "img/container-site-d-48.png",
-    "96": "img/container-site-d-96.png"
+    "16": "img/multiaccountcontainer-16.svg"
   },
   "applications": {
     "gecko": {
-      "id": "@testpilot-containers",
+      "id": "testpilot-containers@mozilla.org",
```

Comment:
>  if this is actually supported by AMO

cc @diox

Reply:
AMO won't even let you upload a new version of an existing add-on with a different extension id.

Reply:
@htwyford I was actually thinking to something like [Bug 1394579](https://bugzilla.mozilla.org/show_bug.cgi?id=1394579), and basically check for the privileged signature (I've been also double-checking with @diox under which criteria AMO would allow us to sign as privileged an update for the multi account container, without changing the id)

Reply:
@htwyford I just wanted to let you know that yesterday I attached a patch to [Bug 1394579](https://bugzilla.mozilla.org/show_bug.cgi?id=1394579) and it shouldn't be far from landing in mozilla-central (the patch already got a review pass and today I added some testing coverage to it).

Once Bug 1394579 will reach nightly, you should be able to proceed with this PR without having to change the addon id.

To be allowed to use the non standard SVG context-fill, the multi-account-container will have to be signed with the "Mozilla extensions" certificate if it wasn't already, but during development you can simulate that by:
- using a non release Firefox build
- set the "extensions.experiments.enabled" about:config pref  to true
- and install the extension temporarily

Reply:
@htwyford we have been discussing more and evaluating what would be the more reasonable way to unblock this PR, and after having heard all the various perspective we are convinced that Bug 1394579 (which has been landed in mozilla-central today) is not yet the best path for multi-account-containers ([1]).

We have determined a different approach that should allow to also unblock this PR without requiring any changes besides the one you have already prepared in this PR, and we are reaching the necessary agreement with the team that maintains that part of Gecko. The new approach should become part of [Bug 1710917](https://bugzilla.mozilla.org/1710917) and so you may follow that bugzilla issue to be notified when this PR can be finally unblock.

-----

[1]: it is technically possible as described in my last comment, unlike changing the addon id, but there is a considerable amount of work to apply all the necessary change to this extension to fit into the different process that is required to get that type of xpi signature